agent/stats/engine_unix_integ_test.go

```diff
@@ -32,3 +55,95 @@ func TestStatsEngineWithNetworkStatsDifferentModes(t *testing.T) {
 		testNetworkModeStats(t, tc.NetworkMode, tc.StatsEmpty)
 	}
 }
+
+func TestStatsEngineWithServiceConnectMetrics(t *testing.T) {
+	// Create a new docker stats engine
+	engine := NewDockerStatsEngine(&cfg, dockerClient, eventStream("TestStatsEngineWithExistingContainersWithoutHealth"))
```

Comment:
Should the event stream name be TestStatsEngineWithServiceConnectMetrics ?

Reply:
oops totally missed it!! I will update it.